### DISCUSSION TITLE
Scratch UI Redesign

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#5116] [Ideas]
Is it feasable to have the entire front page of Scratch just overhauled by a single addon. I found this project on Scratch by an aspiring UI designer https://scratch.mit.edu/projects/719334055 and have started work on making a demo at https://proton15.github.io/ScratchUI/ and was wondering how to make it work as an addon. Thanks!

**Comment by BroJac5246** [comment]
If you want, you can check out our guide on creating addons at [https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/). Do note, however, that such an overhaul would be _unlikely_ to be added as an addon. You can still make it, release it, or open an issue and create a pull request for it, it just might not be added. Hope this helps :)

**Reply by ghost** [reply]
Ok, thanks. I've made extensions for Scratch Addons before, they were just buggy and never got released.

**Comment by Secret-chest** [comment]
I suggest you just release it as a separate extension / userstyle / userscript.

**Comment by Secret-chest** [comment]
> aspiring UI designer

Yeah, made in paint editor with Google Drawings SVG text. How would you make it compatible tho? Only 6 projects on profile???

**Reply by ghost** [reply]
Just by using all the same IDs and classes. I only have 6 projects because I'm moving from an old Scratch and GitHub account.
